rampwf/utils/submission.py

```diff
@@ -320,9 +320,10 @@ def bag_submissions(problem, cv, y_train, y_test, predictions_valid_list,
     print_title('----------------------------')
     score_type_index = (slice(None) if score_type_index is None
                         else score_type_index)
-    score_type = problem.score_types[score_type_index]
-    score_type = ([score_type] if not isinstance(score_type, Iterable)
-                  else score_type)
+    score_types = problem.score_types[score_type_index]
+    score_types = (
+        [score_types] if not isinstance(score_types, Iterable)
+        else score_types)
```

Comment:
You should either let it as it is or use

```python
    score_types = (
        [score_types]
        if not isinstance(score_types, Iterable)
        else score_types
)
```
if you want to use black or yapf style

Reply:
OK.